### DISCUSSION TITLE
[회의록] : 6주차 이주영

### DISCUSSION THREAD

**Discussion by 2zzzyoung** [FE3-part4-team2/-Nomad#113] [회의록]
### 📓 로그인/회원가입 페이지
엔터 누르면 버튼 클릭되게

### 📓 메인 페이지

- [ ] 메인 페이지 무한스크롤 리액트 쿼리로 수정(율민님이 hook 추가해놓으심),
- [x] 별점 아이콘과 평점 높이 맞추기,
- [x] 드롭다운 리스트 줄바꿈 문제,
- [x] 카테고리 분류 줄바꿈 문제,
- [x] 검색창 아이콘 수정,
- [x] 모든 체험 카테고리 추가(가격순으로 보다가 최신순으로 보고 싶을 때),
- [x] 배너 클릭했을 때 해당 페이지로 이동하게,

### 📓 체험 상세 페이지

- [x] 상세 페이지 이미지 레이아웃 변경,
- [x] 예약 불가능 날짜 선택했을 때  예약 시간에 ‘선택 가능한 시간이 없습니다.’,
- [x] 달력 다른 달 데이터도 가져오게 수정,
- [x] 화면 스크롤 버튼 아래로,
- [x] 댓글 하나만 길게 보이는 현상,
- [x] 카카오맵 뜨는지 확인,
- [x] 예약 가능한 시간 많아지면 길게 넘어가는 부분 수정,
- [x] 캘린더 회색 부분 없애기,
- [x] 캘린더 확대하면 두 줄로 보임,

### 📓 내 체험 관리

- [ ] 후기 작성 모달,
- [ ] 내 체험 관리 삭제 안되는데 리스트에서 사라짐,
- [ ] 내 체험 관리 케밥 위치 변경(오른쪽으로),
- [ ] 내 체험 관리 이미지 클릭하면 이동하게,
- [ ] 타이틀이 길어지면 …으로 보이게,

### 📓 내 체험 등록

- [x] 내 체험 등록/수정 페이지의 등록/수정하기 버튼을 아래로,
- [x] 내 체험 등록 주소 인풋과 버튼 사이에 갭 주기,
- [ ] 내 체험 등록 배너 이미지 추가하면 +이미지 등록 사라지게,
- [ ] 내 체험 등록 예약 시간 분 단위가 안 보이는 현상 수정,
- [ ] 내 체험 등록 예약 시간 시계 아이콘 말고 인풋 자체를 누르면 시간 수정할 수 있게,
- [ ] 시간이 현재 시간으로 뜨는데 디폴트 값을 오후 12시로 고정되게,

### 📓 내 정보
Div 안에 인풋을 넣은거 빼주기,

### 📓 내 체험 수정
체험 수정 이미지 추가 안하면 수정 안되는 문제(해결되는건지??),

### 📓 예약 현황

- [x] 예약 현황 페이지 예약 정보 모달이 fixed로 되어있어 스크롤 따라가는 문제 수정,
- [x] 알람 모달 오른쪽 여백 더 주기,
- [x] 알람 없을 때 기본 아이콘 추가,

#### 📓 내 정보 컴포넌트 내 정보/예약 내역 누르면 호버되게.

기능 먼저 끝내고 반응형 안 한 사람은 반응형 작업하기!

### 📘 추가하면 좋을 것 같은 기능
찜/북마크 기능,
최근 본 페이지,

## 📌 앞으로 미팅은 주 1회 화요일 저녁 10시

**Comment by yulmai999** [comment]
다크모드 추가 예정
